bpmn-model/src/test/java/io/zeebe/model/bpmn/util/time/TimeDateTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.model.bpmn.util.time;
+
+import static io.zeebe.model.bpmn.util.time.TimeDate.parse;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import org.junit.Test;
+
+public class TimeDateTest {
+  @Test
+  public void shouldParseTimeDateCorrectly() {
+    // given
+    final ZonedDateTime currentTime = ZonedDateTime.now();
+
+    // when
+    final Timer timeDate = TimeDate.parse(currentTime.toString());
+
+    // then
+    assertThat(timeDate.getRepetitions()).isEqualTo(1);
+    assertThat(timeDate.getDueDate(0)).isEqualTo(currentTime.toInstant().toEpochMilli());
+  }
+
+  @Test
+  public void shouldRejectTimeCycle() {
```

Comment:
Hm, actually I'm not sure if this test is really important/useful anymore...what do you think?